apps/studio/components/interfaces/Organization/GeneralSettings/OptInToOpenAIToggle.tsx

```diff
@@ -27,28 +27,22 @@ export const OptInToOpenAIToggle = () => {
           className="flex flex-col gap-y-4 text-sm text-foreground-light"
         >
           <p>
-            Supabase AI utilizes Amazon Bedrock ("Bedrock"), a service designed with a strong focus
-            on data privacy and security.
+            Supabase AI utilizes third-party AI providers designed with a strong focus on data
+            privacy and security.
           </p>
 
           <p>
-            Amazon Bedrock does not store or log your prompts and completions. This data is not used
-            to train any AWS models and is not distributed to third parties or model providers.
-            Model providers do not have access to Amazon Bedrock logs or customer prompts and
-            completions.
+            By default, no information is shared with third-party AI providers unless you explicitly
+            provide consent. With your permission, Supabase may share customer-generated prompts,
+            database schema, database data, and project logs with these providers. This information
+            is used solely to generate responses to your queries and is not retained by the
+            providers or used to train their foundation models.
```

Comment:
Are we intentionally using the word "foundation" here? Wondering if this will cause unwarranted worry that it will still train any "other" models, which is not true.